great_tables/_formats.py

```diff
@@ -26,7 +27,7 @@
     to_list,
     _get_column_dtype,
 )
-from ._text import _md_html
+from ._text import _md_html, escape_pattern_str_latex
```

Comment:
The new function `escape_pattern_str_latex()` is used here for escaping string literals in the `pattern=` arg. Only LaTeX for now, we could later generalize to include the `"html"` context later.